torch/testing/_internal/common_device_type.py

```diff
@@ -4,6 +4,7 @@
 import runpy
 import threading
 from functools import wraps
+import typing
```

Comment:
using `from typing import Any, ...` is better since you dont need to type `typing.*` below.